frontend/src/components/GroupCard/GroupCard.tsx

```diff
@@ -47,6 +47,9 @@ export const GroupCard: React.FC<GroupCardProps> = (props) => {
           <Skeleton colorMode="light">
             <H4 style={h4}>{props.Name}</H4>
           </Skeleton>
+          <Skeleton colorMode="light">
+            <P style={bodyNormal}>{props.Criteria}</P>
```

Comment:
The style is not bodyNormal.

Reply:
I don't know the information about the style. Currently, I changed to bodyLarge. Please let me know if it it is not correct.

Reply:
We can go through this during today's meeting. :)